doc/api/policy.md

```diff
@@ -0,0 +1,51 @@
+# Policies
+
+<!--introduced_in=TODO-->
+<!-- type=misc -->
+
+> Stability: 1 - Experimental
+
+<!-- name=policy -->
+
+Node.js contains experimental support for creating policies on loading code.
+
+Policies are a security feature intended to allow guarantees
+about what code Node.js is able to load. The use of policies assumes
+safe practices for the policy files such as ensuring that policy
+files cannot be overwritten by the Node.js application by using
+file permissions.
+
+## Enabling
+
+<!-- type=misc -->
+
+The `--experimental-policy` flag can be used to enable features for policies when loading modules.
+
+Once this has been set, all modules must conform to a policy file passed to the flag:
+
+```sh
+node --experimental-policy=policy.json app.js
+```
+
+## Features
+
+### Integrity Checks
+
+Policy files must use integrity checks with Subresource Integrity strings
+compatible with the browser
+[integrity attribute](https://www.w3.org/TR/SRI/#the-integrity-attribute)
+associated with absolute URLs.
```

Comment:
I would add a warning to detail the behavior: "If a policy check fails, the call ro `require` will throw an exception"

Reply:
that is in https://github.com/nodejs/node/pull/23834/files#diff-a85ad5bb9b9ea49ac5513cb0014e3cfeR52